tests/foreman/ui_airgun/test_repository.py

```diff
@@ -123,3 +123,56 @@ def test_positive_create_as_non_admin_user(module_org, test_name):
         )
         assert session.repository.search(
             product.name, repo_name)[0]['Name'] == repo_name
+
+
+@tier2
+@upgrade
+def test_positive_discover_repo_via_existing_product(session, module_org):
+    """Create repository via repo-discovery under existing product
+
+    :id: 9181950c-a756-456f-a46a-059e7a2add3c
+
+    :expectedresults: Repository is discovered and created
+
+    :CaseLevel: Integration
+    """
+    repo_name = 'fakerepo01'
+    product = entities.Product(organization=module_org).create()
+    with session:
+        session.organization.select(org_name=module_org.name)
```

Comment:
there's definitely no need to do that for default user & module org, that's basically the idea of default org to avoid explicitly selecting organization in every single test. 'just in case' isn't a good enough reason - then 99% of tests will fail in case of issue with default org, except of these ones, so there's no point in selecting org in few specific tests.